static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -33,64 +34,70 @@ define([
-        };
+        }
 
-        if (commercialFeatures.dfpAdvertising) {
-            var initialTag = dfpEnv.sonobiEnabled ? setupSonobi() : Promise.resolve();
-            return initialTag.then(setupAdvertising).catch(function(){
-                // A promise error here, from a failed module load,
-                // could be a network problem or an intercepted request.
-                // Abandon the init sequence.
-                return fastdom.write(removeAdSlots);
-            });
+        function moduleCompleted() {
+            performanceLogging.moduleEnd(moduleName);
         }
 
-        return fastdom.write(removeAdSlots);
-    }
+        function setupAdvertising() {
+            // Use Custom Timing to time the googletag code without the sonobi pre-loading.
+            performanceLogging.moduleStart(moduleName);
 
-    function setupSonobi() {
-        return new Promise.resolve(require(['js!sonobi.js']));
-    }
+            return new Promise(function(resolve) {
 
-    function setupAdvertising() {
+                if (dfpEnv.sonobiEnabled) {
+                    // Just load googletag. Sonobi's wrapper will already be loaded, and googletag is already added to the window by sonobi.
+                    require(['js!googletag.js']);
+                    performanceLogging.addTag('sonobi');
+                } else {
+                    require(['js!googletag.js']);
 
-        return new Promise(function(resolve) {
+                    if (dfpEnv.prebidEnabled) {
+                        dfpEnv.prebidService = new PrebidService();
+                        performanceLogging.addTag('prebid');
+                    } else {
+                        performanceLogging.addTag('waterfall');
+                    }
+                }
 
-            if (dfpEnv.sonobiEnabled) {
-                // Just load googletag. Sonobi's wrapper will already be loaded, and googletag is already added to the window by sonobi.
-                require(['js!googletag.js']);
-                ophanTracking.addTag('sonobi');
-            } else {
-                require(['js!googletag.js']);
+                window.googletag.cmd.push = raven.wrap({deep: true}, window.googletag.cmd.push);
 
-                if (dfpEnv.prebidEnabled) {
-                    dfpEnv.prebidService = new PrebidService();
-                    ophanTracking.addTag('prebid');
-                } else {
-                    ophanTracking.addTag('waterfall');
-                }
-            }
+                window.googletag.cmd.push(
+                    setListeners,
+                    setPageTargeting,
+                    moduleCompleted,
+                    resolve
+                );
+            });
+        }
 
-            window.googletag.cmd.push = raven.wrap({deep: true}, window.googletag.cmd.push);
+        if (commercialFeatures.dfpAdvertising) {
+            return prepareSonobiTag.init().then(setupAdvertising).catch(function(){
```

Comment:
Hasn't `prepareSonobiTag` already been called at this point? (in `commercial.js`)